digicampipe/image/utils.py

```diff
@@ -0,0 +1,416 @@
+from astropy.io import fits
+from astropy.stats import sigma_clipped_stats
+import numpy as np
+import scipy
+from skimage.draw import polygon
+import matplotlib.pyplot as plt
+from time import clock
+import cv2
+from scipy import signal, ndimage, optimize
+import numbers
+from photutils import DAOStarFinder
+
+
+def crop_image(image, crop_pixel1, crop_pixel2):
```

Comment:
I would propose to make this a 2-parameter function with this interface `image, rectangle`.

Reply:
fixed in 4724d7158293